Auth0Tests/OAuth2GrantSpec.swift

```diff
@@ -221,11 +215,25 @@ class OAuth2GrantSpec: QuickSpec {
                 }
             }
 
-            it("shoud fail credentials no nonce") {
-                pkce = PKCE(authentication: authentication, redirectURL: redirectURL, verifier: verifier, challenge: challenge, method: method, responseType: response)
+            it("should fail with no nonce") {
+                pkce = PKCE(authentication: authentication, redirectURL: redirectURL, verifier: verifier, challenge: challenge, method: method, responseType: response, nonce: nil, leeway: leeway)
+                let token = UUID().uuidString
+                let code = UUID().uuidString
+                let values = ["code": code, "id_token": idToken]
+                stub(condition: isToken(domain.host!) && hasAtLeast(["code": code, "code_verifier": pkce.verifier, "grant_type": "authorization_code", "redirect_uri": pkce.redirectURL.absoluteString])) { _ in return authResponse(accessToken: token, idToken: idToken) }.name = "Code Exchange Auth"
+                waitUntil { done in
+                    pkce.credentials(from: values) {
+                        expect($0).to(beFailure())
+                        done()
+                    }
+                }
+            }
+            
+            it("should fail with an invalid id_token") {
+                pkce = PKCE(authentication: authentication, redirectURL: redirectURL, verifier: verifier, challenge: challenge, method: method, responseType: response, nonce: nil, leeway: leeway)
                 let token = UUID().uuidString
                 let code = UUID().uuidString
-                let values = ["code": code, "id_token" : idToken]
+                let values = ["code": code, "id_token": generateJWT(iss: nil).string, "nonce": nonce]
```

Comment:
pass the default `nonce` value you defined on the top of the file to the generator

Reply:
Done in ca28451be12118c8928745aead45feaa63e669e1.